selfdrive/car/gm/carcontroller.py

```diff
@@ -137,17 +143,17 @@ def update(self, enabled, CS, frame, actuators, \
       else:
         apply_gas = int(round(interp(final_pedal, P.GAS_LOOKUP_BP, P.GAS_LOOKUP_V)))
         apply_brake = int(round(interp(final_pedal, P.BRAKE_LOOKUP_BP, P.BRAKE_LOOKUP_V)))
+      if not self.car_fingerprint in NO_ASCM_CARS:
```

Comment:
This check does the same thing is the `if self.car_fingerprint not in SUPERCRUISE_CARS:` at line 130.

At the very least I think this if statement should go away and we should do something like.

`if self.car_fingerprint not in SUPERCRUISE_CARS and self.car_fingerprint not in NO_ASCM_CARS:`

Long term its probably better to have new car parameter 'LongitudinalControl' and just check that in the if statement for if we are going to do gas/brake commands.